### PR TITLE
Fixing fix_test/test_target_is_device_ptr.c to use the headerfile

### PR DESCRIPTION
**Pull request by josemonsalve2**
This is to fix the `test_target_is_device_ptr.c` test to use the headerfile

working with @tmh97

### CODE REVIEW

**Comment by josemonsalve2** [comment]
Tackling issue #10

**Comment by josemonsalve2** [comment]
TEsted on summit it passes in all the compilers

File: tests/4.5/target/test_target_is_device_ptr.c [hunk 1]
```diff
@@ -1,52 +1,54 @@
+//===--- test_target_is_device_ptr.c-----------------------===//
```
Comment (jhdavis8) [inline tests/4.5/target/test_target_is_device_ptr.c]: (coding style fix) Top line of header is short, should match length of bottom line.

File: tests/4.5/target/test_target_is_device_ptr.c [hunk 2]
```diff
@@ -1,52 +1,54 @@
...
+  omp_target_free(array_device, omp_get_default_device());
   free(array_host);
 
-  if (!errors)
-    printf("Test passed on %s\n", (isHost ? "host" : "device"));
-  else
-    printf("Test failed on %s\n", (isHost ? "host" : "device"));
-
-  return errors;
+  OMPVV_REPORT_AND_RETURN(errors)
```
Comment (tmh97) [inline tests/4.5/target/test_target_is_device_ptr.c]: add ; to end of ompvv_report_and_return macro

**Pull request merged**